server/src/controller/producto.ts

```diff
@@ -1,3 +1,4 @@
+import { logger } from '../util/logger'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/c0dejohn/e-commerce-backend/pullRequest?prid=8292833)
```suggestion
import { logger } from "../util/logger"
```